src/platform/ios.rs

```diff
@@ -85,6 +85,21 @@ pub trait WindowExtIOS {
     /// [`setNeedsStatusBarAppearanceUpdate()`](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621354-setneedsstatusbarappearanceupdat?language=objc)
     /// is also called for you.
     fn set_preferred_status_bar_style(&self, status_bar_style: StatusBarStyle);
+
+    /// Sets whether the [`Window`] should recognize pinch gestures.
+    ///
+    /// The default is to not recognize gestures.
+    fn recognize_pinch_gesture(&self, should_recognize: bool);
+
+    /// Sets whether the [`Window`] should recognize double tap gestures.
+    ///
+    /// The default is to not recognize gestures.
+    fn recognize_doubletap_gesture(&self, should_recognize: bool);
+
+    /// Sets whether the [`Window`] should recognize rotation gestures.
+    ///
+    /// The default is to not recognize gestures.
+    fn recognize_rotation_gesture(&self, should_recognize: bool);
```

Comment:
That the events being off by default should be documented in the `WindowEvent` enum.

Reply:
Also, do you have a use-case for controlling this after window creation time? If not, perhaps these methods should be on `WindowBuilderExtIOS` instead?

Reply:
> do you have a use-case for controlling this after window creation time?

I think so, enabling gesture recognition delays basic touch a bit, that seemed not desirable when it's not needed. Coming from Bevy and games, I probably don't need gestures on my menu.